docs/installation.adoc

```diff
@@ -63,7 +63,7 @@ $ brew install kadel/odo/odo
 ==== Tarball installation
 
 ----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-darwin-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
+# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-darwin-amd64.tar.gz | tar -xvzf odo-darwin-amd64.tar.gz -C /usr/local/bin; chmod +x /usr/local/bin/odo'
```

Comment:
@amitkrout this change

Reply:
https://github.com/openshift/odo/pull/2009#discussion_r316532573